examples/using_smirnoff_with_amber_protein_forcefield/toluene_in_T4_lysozyme.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
     "### Parametrize a molecule with smirnoff99Frosst\n",
     "\n",
-    "First, we parametrize the ligand (toluene) with the smirnoff99Frosst force field through the usual route"
+    "First, we parametrize the ligand (toluene) with the smirnoff99Frosst force field through the usual route to create an OpenMM `System`."
```

Comment:
One of the subtle points of this example is that the toluene coordinates are already compatible with the protein coordinates. Maybe add another copy of `Note that the coordinates of protein and ligand in this example are determined by the PDB file, and they are already consistent with the ligand being positioned in the binding pocket` here.

Reply:
Will do!